src/main/scala/scala/concurrent/stm/ccstm/Stats.scala

```diff
@@ -25,7 +25,7 @@ private[ccstm] object Stats {
     def toStr(k: A): String = k.toString
 
     def contents: Seq[(String, Long)] = {
-      val aa = _counters.entrySet.toSeq map { e => (toStr(e.getKey) -> e.getValue.apply()) }
```

Comment:
This might be even improved performance-wise to `asScala.iterator.map(...).toSeq`?